tests/unit/src/test/scala/tests/RangeFormattingSuite.scala

```diff
@@ -62,7 +62,7 @@ object RangeFormattingSuite extends BaseLspSuite("rangeFormatting") {
   )
 
   check(
-    "paste-on-fist-line",
+    "paste-on-fist-line-with-pipe",
```

Comment:
```suggestion
    "paste-on-first-line-with-pipe",
```

👊

Reply:
Nope, not fixing, it was like that before :trollface: